index.html

```diff
@@ -1411,10 +1421,23 @@
             <li>Return <var>P</var>, but continue running these steps <a>in
             parallel</a>.
             </li>
-            <li>If the user agent is unable to <a>monitor the list of available
-            presentation displays</a> for the entire duration of the
-            <a>controlling browsing context</a> (e.g., because the user has
-            disabled this feature), then:
+            <li>If one of the following conditions is true, run the next
+            substeps:
+              <ul>
+                <li>The user agent is unable to <a>monitor the list of available
```

Comment:
It seems like we should reject P with NotSupportedError here.  Instead of saying there are no displays, there's actually no way for the page to find out whether there are displays are not.